### PR TITLE
[Integration Manager] Move `receiveFormInfo` into the Merlins Connector

### PR DESCRIPTION
**Pull request by jeremywiebe**
Discovered that the `receiveFormInfo` is an implementation detail of the Merlin's connector. Moving it in and removing usage in the Demandware connector.

 **JIRA**: **N/A**
**Depends on**: https://github.com/mobify/platform-scaffold/pull/459 👈  Merge this first
 **Linked PRs**: https://github.com/mobify/platform-scaffold/pull/421

## Changes
- Move `receiveFormInfo` into the Merlins connector 
- Add placeholder `actions.js` for both connectors and hook the into the reducers to allow for connector-specific data in the store.

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Ensure you can go through home -> PLP -> PDP -> Add to cart -> Checkout

### CODE REVIEW

File: web/app/integration-manager/_demandware-connector/actions.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import {createAction} from '../utils/utils'
```
Comment (jnurse) [inline web/app/integration-manager/_demandware-connector/actions.js]: I think you need another `../` here

File: web/app/integration-manager/_merlins-connector/actions.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import {createAction} from '../utils/utils'
```
Comment (jnurse) [inline web/app/integration-manager/_merlins-connector/actions.js]: Same as above. You need to go up one more folder.

**Comment by jnurse** [comment]
@jeremywiebe I'm also getting an error when I switch this over to the demandware connector.

**Comment by jeremywiebe** [comment]
@jnurse This PR is based off of the Merlin's connector PLP branch so the Demandware stuff will not work yet.  What do you think about merging this PR and then I can re-fix it for Demandware branch once that's merged.

**Comment by jnurse** [comment]
👍

**Pull request merged**